### DISCUSSION TITLE
Vote for C's new name!

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#320] [Announcements]
Please remember to vote for C's new name!
You do so by navigating to the [vote page](https://github.com/TodePond/Dreamberd/issues/297).

Let it be known — C **will** be renamed to the winning name upon the conclusion of the vote.

**Note:** The vote will end at midnight on the 9th September 2099.

**Comment by PalmDevs** [comment]
C++ for sure!

**Comment by erictg** [comment]
++C

**Comment by juju2143** [comment]
I vote that we should do this every 2 weeks, for consistency

**Reply by erictg** [reply]
you mean consistent-C right?

**Reply by juju2143** [reply]
I C what you did there

**Comment by HomemadeToast57** [comment]
"C!!!!!!!!!!!!!!" should be the name (notice the use of the '!')

**Comment by chairclr** [comment]
what about C?

**Reply by D-u-c-k-s-e-l** [reply]
Please clarify: `C` or `C?`?

**Reply by chairclr** [reply]
LGTM

**Comment by skifli** [comment]
E, as aforementioned [here](https://github.com/TodePond/C/issues/297#issuecomment-1664252003).

**Comment by elRatto21** [comment]
C--

**Reply by AsynchronousAI** [reply]
C-- exist

**Comment by ninanomenon** [comment]
C++++ or shorter C#

**Comment by AsynchronousAI** [comment]
C++#

**Reply by D-u-c-k-s-e-l** [reply]
So C++++++

**Comment by i-rember** [comment]
C--

**Comment by ghost** [comment]
DreamBerd

**Comment by MelonenBiber** [comment]
Rust

**Comment by TopHatTacco** [comment]
Defiantly C!

**Comment by D-u-c-k-s-e-l** [comment]
++Rust

**Comment by deanveloper** [comment]
we should name it after my favorite programming language (CSS), but it's an extended version so let's call it XSS

**Comment by ghost** [comment]
F

**Comment by 0xrpj** [comment]
C Blunt

**Comment by anonymity44** [comment]
C-Next

**Comment by loglot** [comment]
this is out of date, c was renamed out of time of the vote, it's still going on, but for now all refferences of C should be replaced with WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite

**Reply by ghost** [reply]
That has a nice ring to it.

**Reply by NikolafTheGreat** [reply]
Hey. I am confused. I am very out of the loop. It seems like there is some kind of drama here. What is going on?
I am concerned.

**Reply by loglot** [reply]
Wdym, that's just the name

**Reply by NikolafTheGreat** [reply]
You know what? Forget I asked. I was being paranoid. It is a really nice name.

**Comment by treeplate** [comment]
the link is invalid

**Reply by TodePond** [reply]
fixed

**Comment by dli152** [comment]
C♭

**Comment by James-HoneyBadger** [comment]
CC

**Reply by loglot** [reply]
Nah, it just doesn't hit as hard as c++
